docs/development/modules.md

```diff
@@ -356,6 +356,24 @@ druid.coordinator.cleanupMetadata.duty.killSupervisors.retainDuration=PT0M
 druid.coordinator.cleanupMetadata.period=PT10S
 ```
 
+### Routing data through a HTTP proxy for your extension
+
+It is possible to adds the ability for the HttpClient of your extension to connect through a HTTP proxy. 
+Internally, Druid will augment the channel factory to check if it is supposed to be proxied and, if so, connect to the proxy host first, 
+issue a CONNECT command through to the final recipient host and then give the channel to the normal http client for usage.
+
+To enable this feature and allow for proxy in your extension's HTTP client, 
+first, add `HttpClientProxyConfig` as a `@JsonProperty` to the HTTP config class of your extension. 
+Next, in the extension's module class, add `HttpProxyConfig` to `HttpClientConfig`. For example, 
```

Comment:
```suggestion
To enable proxy connection for your extension's HTTP client:
1.  Add `HttpClientProxyConfig` as a `@JsonProperty` to the HTTP config class of your extension. 
2. In the extension's module class, add `HttpProxyConfig` to `HttpClientConfig`. For example, where `HttpClientConfig` is the HTTP config class for your extension:
```
(Not sure if I interpreted this correctly, but include the information from the note on line 375 in the for example statement here)

Reply:
Done